graphql/mutation_resolver.go

```diff
@@ -329,6 +334,9 @@ func (r *mutationResolver) SetPatchVisibility(ctx context.Context, patchIds []st
 	}
 
 	for _, p := range patches {
+		if !hasEditPatchPermission(user, p) {
+			return nil, Forbidden.Send(ctx, "can't change visibility of another user's patch")
```

Comment:
This technically could be not the problem right? They could just not have the right permission? Should this error message be made more generic?

Reply:
Changed to more generic `"not authorized to change patch visibility"`